packages/tomd/tests/test_gold_standard.py

```
@@ -0,0 +1,176 @@
+
+    def test_has_reply_to(self, d4036_fm):
+        assert "reply-to" in d4036_fm
+        assert isinstance(d4036_fm["reply-to"], list)
+        assert len(d4036_fm["reply-to"]) >= 1
+
+
+class TestD4036Structure:
+    def test_no_h1_in_body(self, d4036_md):
+        fm_end = d4036_md.find("---", 4)
+        body = d4036_md[fm_end + 3:]
+        h1_lines = [l for l in body.split("\n")
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)